### PR TITLE
adds first blog post ft eddy from denver art museum

### PR DESCRIPTION
**Pull request by ablwr**
Requesting posting and deploying! No pix so should be easy.

### CODE REVIEW

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
...DAM we have decided to create MKV/FFV1/LPCM files for any digital video in the collection that uses a proprietary codec, or an obsolete format. A lot of the electronic media in the museum’s [design collection](https://www.google.com/url?q=http://denverartmuseum.org/collections/architecture-design-graphics&sa=D&ust=1497302411729000&usg=AFQjCNHi1BIRCa2vf08SKkJmoxmCX32ARg) comes from [the AIGA Archives](https://www.google.com/url?q=http://designarchives.aiga.org/%23/home&sa=D&ust=1497302411731000&usg=AFQjCNHc2eqfNGphFONluVYYj8ivFAwMPA), which the DAM collects and preserves. A ton of the video files from the AIGA Archives were created in the aughts, and they use all kinds of whacky codecs - my favorite so far is one that MediaInfo identifies as “RoadPizza” (apparently a QuickTime codec). Given that I don’t want to rely on the long-term support of the RoadPizza codec, we’re normalizing files like that to MKV/FFV1/LPCM through an automated Archivematica micro-service that uses the following ffmpeg script (which I cobbled together using [ffmprovisr](https://www.google.com/url?q=https://amiaopensource.github.io/ffmprovisr/&sa=D&ust=1497302411732000&usg=AFQjCNHx6UqnvVMH3B319D-dCYkd6EEYYg)):
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md]: ffmpeg script --> FFmpeg script

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
...ps://www.archivematica.org/en/&sa=D&ust=1497302411701000&usg=AFQjCNGiYD7IxXFq_vJwvx1v9OhMyEjDsQ) for ingest of media into our digital repository. It allows us to automate the creation of METS wrapped PREMIS XML documentation, while manually customizing which microservices we choose to use (or not use) as we ingest new works. Some of the microservices I use on a regular basis are file format identification through [Siegfried](https://www.google.com/url?q=http://www.itforarchivists.com/siegfried&sa=D&ust=1497302411703000&usg=AFQjCNFls8VIadFVVfzr7aMZO4wnAxp8KQ), metadata extraction with [MediaInfo](https://www.google.com/url?q=https://mediaarea.net/ru/mediainfo&sa=D&ust=1497302411704000&usg=AFQjCNE2WXrMudOm6zPEugJ8iAiqmrqCtQ), [Exiftool](https://www.google.com/url?q=http://www.sno.phy.queensu.ca/~phil/exiftool/&sa=D&ust=1497302411705000&usg=AFQjCNGQvM_KgEjvGZxJSdFCmY6FQEWBFA), and [Droid](https://www.google.com/url?q=http://www.dcc.ac.uk/resources/external/droid&sa=D&ust=1497302411706000&usg=AFQjCNGX3FGken7O5d-JRMZ9EnANE6DGCw), and normalization using tools like [ffmpeg](https://www.google.com/url?q=https://ffmpeg.org/&sa=D&ust=1497302411707000&usg=AFQjCNEb6U7fL221fOQdmYIQ7KkZAROX3Q).
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md]: \[Exiftool\] --> \[ExifTool\]
\[ffmpeg\] --> \[FFmpeg\]

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Things couldn’t be more different at LPB. All completed locally produced programming automatically becomes part of the LPB archive. The LPB Archive is then responsible for preserving, describing and making that content accessible through the Louisiana Digital Media Archive (LDMA), located at [http://www.ladigitalmedia.org/](https://www.google.com/url?q=http://www.ladigitalmedia.org/&sa=D&ust=1497302411709000&usg=AFQjCNHbwgFXa_vEoYBlwssXCi1WyF-m9A). LPB’s ingest procedures need to allow for a lot more throughput, but there’s much less variability in the type of files they collect compared to the DAM. With less of a need for manual assessment, LPB uses an automated process to create mediainfo XML files, an MD5 checksum sidecar, and a MediaConch report through a custom watchfolder application that our IT engineer, Adam Richard, developed. That code isn’t publically available unfortunately, but you can see the scripts that went into it on [my github](https://www.google.com/url?q=https://github.com/eddycolloton/LPB_ArchiveProcesses&sa=D&ust=1497302411710000&usg=AFQjCNFHpD6_n8dWf5Gicn-w0iqKZEwWFQ).
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md]: \[my github\] --> \[my GitHub\]
mediainfo --> MediaInfo

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+**At what point in the archival process do you use MediaConch?**
+
+
+Basically pre-ingest for both LPB and the DAM. When using MediaConch as a policy checker, my goal is to make sure we’re not bothering to ingest a file that is not encoded to spec.
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md:37]: IMO it is unclear (spec of what? format or DAM?) as it is a policy check: "LPB/DAM spec"?

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
...h.264 encoded files in a .mp4 wrapper. You can find the policies I created for LPB [here](https://www.google.com/url?q=https://github.com/eddycolloton/MediaConch&sa=D&ust=1497302411720000&usg=AFQjCNGAbyEM1vzB7hJkVhtTkYVLDqvNEA), and I talk through the rationale of creating those policies in the digital preservation plan that I created for LPB, [available here](https://www.google.com/url?q=https://aapbndsr.files.wordpress.com/2017/03/lpb_digital_preservation_plan_aapb_ndsr.pdf&sa=D&ust=1497302411721000&usg=AFQjCNElVoA6Px0syynhoms4_AdmpOvB1w) (MediaConch stuff on page 24, and page 45). I’m happy to report that LPB is currently testing a new workflow that will transcode uncompressed .mov files into lossless MKV/FFV1/LPCM files (borrowing heavily from the Irish Film Archive’s [lossless transcoding procedures](https://www.google.com/url?q=https://github.com/kieranjol/IFIscripts/blob/master/makeffv1.py&sa=D&ust=1497302411722000&usg=AFQjCNHbDuGNzix5cDWsLOm6m9ccLT1YIA), as well as the CUNY TV team’s [“make lossless” microservice](https://www.google.com/url?q=https://github.com/mediamicroservices/mm/blob/master/makelossless&sa=D&ust=1497302411724000&usg=AFQjCNGC5RRaNQ_h1pnbNukiiKlcY1wDNA)).
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md]: h.264 --> H.264

File: _posts/2017-06-13-interview-with-eddy-colloton.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+# Interview with Eddy Colloton
+
+*Editor's note: This is the first in a series of interviews with people using MediaConch within their institutions. Enjoy!*
+
+
+**Hey Eddy! Introduce yourself please.**
+
+Hey Ashley! I’ve recently become a “Denverite” and have started a new job as an [Assistant Conservator specializing in electronic media](https://www.google.com/url?q=http://www.kunc.org/post/built-past-floppy-disk-and-vhs-art-need-creative-conservation&sa=D&ust=1497302411695000&usg=AFQjCNEd4gxsXMGMqdLlhnkyQ3U_cuCpug) at the Denver Art Museum (DAM). Before that, I was down in Baton Rouge, Louisiana working as a [National Digital Stewardship Resident](https://www.google.com/url?q=https://ndsr.americanarchive.org/&sa=D&ust=1497302411696000&usg=AFQjCNFAxhriwjXWqTbSm04mdYPhmQx3aA) with Louisiana Public Broadcasting (LPB). When I’m not working, I like to listen to podcasts, read comics, play bass, and stare into the endless void that is “twitter” ([@EddyColloton](https://twitter.com/EddyColloton)). I’m on a big H. P. Lovecraft kick right now so let me know if you have any recommendations (Dunwich Horror is my current fav, but At the Mountains of Madness is a close second).
```
Comment (JeromeMartinez) [inline _posts/2017-06-13-interview-with-eddy-colloton.md]: "https://www.google.com/url?q=" and all including and after "&" in the URL are not needed (slower due to redirection, more private info provided to Google), should be removed everywhere.

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]

**Comment by JeromeMartinez** [comment]
@ablwr  you forgot to remove "&" and all after it, when removing google.com

**Comment by ablwr** [comment]
Oh damn it. I am never using Google Docs again.

**Review by JeromeMartinez — APPROVED** [review]

**Pull request merged**